querydsl-jpa-codegen/src/test/java/com/querydsl/jpa/codegen/HibernateDomainExporterTest.java

```diff
@@ -13,203 +13,50 @@
-                                    file.getName()), Charsets.UTF_8);
-            if (!result1.equals(result2)) {
-                System.err.println(file.getName());
-                failures.add(file.getName());
-            }
-        }
-
-        if (!failures.isEmpty()) {
-            fail("Failed with " + failures.size() + " failures");
-        }
-
-        CompileUtils.compile("target/gen4");
-
-    }
-
-    @Test
-    public void Execute_Store() throws IOException {
-        FileUtils.delete(new File("target/gen5"));
-        File contact = new File("src/test/resources/store.hbm.xml");
-        Configuration config = new Configuration();
-        config.addFile(contact);
-        HibernateDomainExporter exporter = new HibernateDomainExporter("Q",
-                new File("target/gen5"), config);
-        exporter.execute();
-
-        File targetFile = new File("target/gen5/com/querydsl/jpa/domain3/QStore.java");
-        assertContains(targetFile, "StringPath code", "StringPath address");
-
-        targetFile = new File("target/gen5/com/querydsl/jpa/domain3/QHardwareStore.java");
-        assertContains(targetFile, "StringPath code = _super.code;", "StringPath address");
-
-        CompileUtils.compile("target/gen5");
-    }
-
-    @Test
-    public void Execute_CompositeKey() throws IOException {
-       // See https://github.com/querydsl/querydsl/issues/1459
-
-        FileUtils.delete(new File("target/gen18"));
-
-        Configuration config = new Configuration();
-        config.addFile(new File("src/test/resources/route1.hbm.xml"));
-        config.addFile(new File("src/test/resources/route2.hbm.xml"));
-        HibernateDomainExporter exporter = new HibernateDomainExporter("Q",
-                new File("target/gen18"), serializerConfig, config);
-        exporter.execute();
-
-        CompileUtils.compile("target/gen18");
-    }
-
-    private static void assertContains(File file, String... strings) throws IOException {
-        assertTrue(file.getPath() + " doesn't exist", file.exists());
-        String result = Files.toString(file, Charsets.UTF_8);
-        for (String str : strings) {
-            assertTrue(str + " was not contained", result.contains(str));
+        HibernateDomainExporter exporter = new HibernateDomainExporter(folder.getRoot(), cfg);
+        exporter.execute();
+
+        File origRoot = new File("../querydsl-jpa/target/generated-test-sources/java");
+        Set<File> files = exporter.getGeneratedFiles();
+        assertFalse(files.isEmpty());
+        for (File file : files) {
+            String path = file.getAbsolutePath().replace(
+                    folder.getRoot().getAbsolutePath(), origRoot.getAbsolutePath());
+            String reference = Files.toString(new File(path), Charsets.UTF_8);
+            String content = Files.toString(file, Charsets.UTF_8);
+            errors.checkThat(content, is(equalTo(reference)));
```

Comment:
Oh it seems we're on an older version of junit than I usually am.
I am at `junit-4.12` and use ErrorCollector myself, so thought that the `reason` would make more sense.
You can either bump the junit version (and see which surefire version works) or just leave it like this.